README.md

```diff
@@ -22,6 +22,8 @@ There are a number of features unique to this module that make it attractive for
 
 1. Modify the variables at the top of `packer/vault.json` to reflect your infrastructure
 1. From the Packer directory run `packer build vault.json`
+1. Make sure, you have an AWS SSH Keypair for the Vault instances. This module doesn't handle it. See [#3](https://github.com/avantoss/vault-infra/pull/3) for details.
+1. Make sure to create a [VPC Endpoint for Amazon S3](https://aws.amazon.com/blogs/aws/new-vpc-endpoint-for-amazon-s3/), [VPC Endpoint for DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/vpc-endpoints-dynamodb.html), and [VPC Endpoint for KMS](https://docs.aws.amazon.com/kms/latest/developerguide/kms-vpc-endpoint.html) in your VPC. These endpoints are required for an instance to be able to communicate with S3 and DynamoDB as Vault's backends and be able to communicate with KMS for auto-unseal feature. S3. See [#3](https://github.com/avantoss/vault-infra/pull/3) for details.
```

Comment:
I like these changes. Suggesting adding one note that VPC endpoints are only required (but always a good idea) if your vault EC2 instances don't have access to the public internet.
```suggestion
1. Make sure to create a [VPC Endpoint for Amazon S3](https://aws.amazon.com/blogs/aws/new-vpc-endpoint-for-amazon-s3/), [VPC Endpoint for DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/vpc-endpoints-dynamodb.html), and [VPC Endpoint for KMS](https://docs.aws.amazon.com/kms/latest/developerguide/kms-vpc-endpoint.html) in your VPC. If you're vault instances cannot access the internet these endpoints are required for an instance to be able to communicate with S3 and DynamoDB as Vault's backends and be able to communicate with KMS for auto-unseal feature. S3. See [#3](https://github.com/avantoss/vault-infra/pull/3) for details.
```